tests/pytorch/test_pytorch_model_export.py

```diff
@@ -769,6 +790,41 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+def compare_state_dicts(state_dict1, state_dict2):
```

Comment:
Can we rename this function to `state_dict_equal` to clarify this function checks the equality of state dicts? The word `compare` sounds a bit ambiguous.